README.md

```diff
@@ -104,7 +104,7 @@ This applies to each `INVERTER[X]` section. X is the number of Inverter starting
 
 | Config value        | Explanation   |
 |-------------------- | ------------- |
-| Phase | which Phase L1, L2, L3 to show|
+| Phase | which Phase L1, L2, L3 to show; use 3P to split power equally over three phases (i.e. Hoymiles micro-inverters only report total power, not seperated by phase)|
```

Comment:
That is not quite right. Only 3-phase inverters report it this way. Just mention this, please

Reply:
What part exactly is 'not quite right'?

Reply:
What you describe is correct. Actually, I meant by this that the path here describes which phase the inverter is connected to. Only 3 phase Hoymiles inverter could have the flag "3P" since we do not support any other Inverters here. I think it is confusing to describe the effect here when you actually want to tell the system that we have a 3 Phase inverter. You could add an explanation under the table (which is much appreciated). Sorry for any confusion (I need more coffee)